CoreJournal/CoreJournal/EntryController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  EntryController.swift
+//  CoreJournal
+//
+//  Created by Jocelyn Stuart on 2/11/19.
+//  Copyright © 2019 JS. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class EntryController {
+    
+    func saveToPersistentStore() {
+        do {
+            let moc = CoreDataStack.shared.mainContext
+            try moc.save() // Save the task to the persistent store.
+        } catch {
+            NSLog("Error saving managed object context: \(error)")
+        }
+    }
+    
+    func loadFromPersistentStore() -> [Entry] {
+        
+        let fetchRequest: NSFetchRequest<Entry> = Entry.fetchRequest()
+        
+        // We could say what kind of tasks we want fetched.
+        
+        let moc = CoreDataStack.shared.mainContext
+        
+        do {
+            return try moc.fetch(fetchRequest)
+        } catch {
+            NSLog("Error fetching tasks: \(error)")
+            return []
+        }
+        
+    }
+    
+    let dateFormatter = DateFormatter()
+    
+    var entries: [Entry] {
+        return loadFromPersistentStore()
+    }
+    
+   func create(withTitle title: String, andBody bodyText: String){
+    _ = Entry(title: title, bodyText: bodyText, timestamp: Date(), identifier: UUID().uuidString)
+        saveToPersistentStore()
+    }
+    
+    func update(withEntry entry: Entry, andTitle title: String, andBody bodyText: String) {
+        guard let index = entries.index(of: entry) else { return }
+        
+        dateFormatter.dateStyle = .short
+        dateFormatter.timeStyle = .short
```

Comment:
You can use `DateFormatter` if you need to convert a String to Date or vice-versa. Otherwise you don't need it, especially in here